src/include/fuzzyduck.hpp

```diff
@@ -10,8 +10,11 @@
 
 #include "duckdb.hpp"
 #include "duckdb/parser/query_node.hpp"
+#include "random_nums_config.hpp"
 
 namespace duckdb {
+using namespace std;
```

Comment:
we try to avoid adding extra namespaces. So you can just prefix most things with `std::` if needed